app/views/components/docs/filter_summary.yml

```diff
@@ -0,0 +1,38 @@
+name: Filter summary
+description: Displays a summary of applied filters as links that can remove a specific filter. Also a link for clearing all filters.
+uses_component_wrapper_helper: true
+accessibility_criteria: |
+  The component items must:
+
+  - accept focus
+  - be focusable with a keyboard
+  - be usable with a keyboard
+  - be usable with touch
+  - indicate when it has focus
+examples:
+  default:
+    data:
+      clear_all_href: "/clear_all_url"
```

Comment:
We might want to make the links in this file all `#` so you can click on them in the component guide without being taken to a random 404